README.md

```diff
@@ -1,15 +1,15 @@
 # go-github #
 
 [![go-github release (latest SemVer)](https://img.shields.io/github/v/release/google/go-github?sort=semver)](https://github.com/google/go-github/releases)
-[![GoDoc](https://img.shields.io/static/v1?label=godoc&message=reference&color=blue)](https://pkg.go.dev/github.com/google/go-github/v65/github)
+[![Go Reference](https://img.shields.io/static/v1?label=godoc&message=reference&color=blue)](https://pkg.go.dev/github.com/google/go-github/v65/github)
 [![Test Status](https://github.com/google/go-github/workflows/tests/badge.svg)](https://github.com/google/go-github/actions?query=workflow%3Atests)
 [![Test Coverage](https://codecov.io/gh/google/go-github/branch/master/graph/badge.svg)](https://codecov.io/gh/google/go-github)
 [![Discuss at go-github@googlegroups.com](https://img.shields.io/badge/discuss-go--github%40googlegroups.com-blue.svg)](https://groups.google.com/group/go-github)
 [![CII Best Practices](https://bestpractices.coreinfrastructure.org/projects/796/badge)](https://bestpractices.coreinfrastructure.org/projects/796)
 
 go-github is a Go client library for accessing the [GitHub API v3][].
 
-Currently, **go-github requires Go version 1.13 or greater**.  go-github tracks
+Currently, **go-github requires Go version 1.21 or greater**.  go-github tracks
```

Comment:
So this is interesting because we used to require a minimum of Go 1.13 but at some point we broke that and required a minimum of Go 1.16.

I honestly don't remember now what minimum version of Go is needed but I'm not sure that Go 1.21 is accurate.

Our policy is to officially test the most recent two minor versions of the Go toolchain, so maybe we should just reword it to say that instead of stating an absolute minimum (unless, of course, we can go back and figure out what the absolute minimum needed is).  However, in my many years of maintaining this library, I can't remember once when someone asked to support an older version of the Go toolchain, so maybe this all is moot.

Thoughts?

Reply:
Maybe "go-github tests against Go version 1.21 and greater" here?

Reply:
Changed to "go-github tests against Go version 1.22 and greater".